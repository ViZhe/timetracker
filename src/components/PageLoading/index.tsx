
import { Spin } from 'antd';
import React from 'react';

import styles from './index.module.css';


const PageLoading: React.FC = () => (
  <div className={styles.wrapper}>
    <Spin size="large" />
  </div>
);


export default React.memo(PageLoading);
