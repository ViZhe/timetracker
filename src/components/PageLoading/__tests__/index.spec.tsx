
import { shallow } from 'enzyme';
import React from 'react';

import PageLoading from '../../PageLoading';


describe('<PageLoading />', () => {
  it('should match snapshot', () => {
    const component = shallow(<PageLoading />);

    expect(component).toMatchSnapshot();
  });
});
