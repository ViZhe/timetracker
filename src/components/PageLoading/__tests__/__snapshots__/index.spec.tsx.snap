// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageLoading /> should match snapshot 1`] = `
<div
  className="wrapper"
>
  <Spin
    size="large"
    spinning={true}
    wrapperClassName=""
  />
</div>
`;
